site/content/docs/5.0/components/modal.md

```diff
@@ -892,6 +892,13 @@ Manually opens a modal. **Returns to the caller before the modal has actually be
 myModal.show()
 ```
 
+Also, you can pass related target:
```

Comment:
Also, please mention that the DOM element passed as the first argument can be received in the modal events (as the `relatedTarget` property)